index.bs

```diff
@@ -2288,7 +2290,7 @@ Registering Formal Objections</h3>
 	related to that review.
 
 	Note: [=Formal Objections=] against matter in a [=technical report=]
-	are required to be addressed before <a href="#rec-advance">requesting advancement</a>
+	<a href="#adv-verif-reqs">are expected to be fully addressed</a> before <a href="#rec-advance">requesting advancement</a>
```

Comment:
This isn't quite what the linked section says, although maybe the intent is the same. The text there is
> [Team](https://pr-preview.s3.amazonaws.com/w3c/w3process/pull/803.html#team) verification ... must be withheld if any Process requirements are not met or if there remain any unresolved Formal Objections (including any [upheld](https://pr-preview.s3.amazonaws.com/w3c/w3process/pull/803.html#uphold) by a [Council](https://pr-preview.s3.amazonaws.com/w3c/w3process/pull/803.html#w3c-council) but not yet [fully addressed](https://pr-preview.s3.amazonaws.com/w3c/w3process/pull/803.html#fully-addressed)), or if the document does not adequately reflect all relevant decisions of the [W3C Council](https://pr-preview.s3.amazonaws.com/w3c/w3process/pull/803.html#w3c-council) (or its delegates).

If there hasn't been a W3C council, can an FO be "[fully addressed](https://pr-preview.s3.amazonaws.com/w3c/w3process/pull/803.html#fully-addressed)"? "Unresolved" leaves space for the resolution to be that the WG wants to [manage dissent](https://pr-preview.s3.amazonaws.com/w3c/w3process/pull/803.html#managing-dissent) or [decide by vote](https://pr-preview.s3.amazonaws.com/w3c/w3process/pull/803.html#Votes), but this note now says that the normative wording should be interpreted in a way that potentially introduces the W3C Council process before FPWD, CR, and PR, in addition to its usual timing between PR and REC.

If that was the intent of the wording in https://www.w3.org/2023/Process-20231103/#transition-reqs, this change is fine, but if not, it should say something like

```suggestion
	<a href="#adv-verif-reqs">are expected to be resolved</a> before <a href="#rec-advance">requesting advancement</a>
```

Reply:
> If there hasn't been a W3C council, can an FO be "[fully addressed](https://pr-preview.s3.amazonaws.com/w3c/w3process/pull/803.html#fully-addressed)"?

There can be, if the FO was resolved by consensus. But if it's not, then no, you cannot publish until the Council process run.

Reply:
This was deemed appropriate by consensus during the Process-CG call of 2023-12-13